HW/HW_P1/Task2.py

```diff
@@ -0,0 +1,9 @@
+number = 1904
```

Comment:
look at this implementation
digit_number = input("Enter a four digit number:")

product =  int(str(digit_number)[0]) * int(str(digit_number)[1]) * int(str(digit_number)[2]) * int(str(digit_number)[3])
print(f"{digit_number[0]} * {digit_number[1]} * {digit_number[2]} * {digit_number[3]} = ", product)
print()

revers = int(''.join(reversed(digit_number)))
print(f"{digit_number} in revers ordef = ", revers)
print()

sort = int("".join(sorted(digit_number)))
print(f"{digit_number} sort =", sort)